apps/project/tests/test_apis.py

```diff
@@ -843,6 +843,13 @@ def test_options(self):
         self.authenticate()
         response = self.client.get(url)
         self.assert_200(response)
+        self.assertIn('regions', response.data)
```

Comment:
We need to add project attributes for the union logic to kick in. Let's add that too.